clients/src/main/resources/common/message/ProduceRequest.json

```diff
@@ -37,7 +37,9 @@
   // Version 10 is the same as version 9 (KIP-951).
   //
   // Version 11 adds support for new error code TRANSACTION_ABORTABLE (KIP-890).
-  "validVersions": "0-11",
+  //
+  // Version 12 is the same as version 10 (KIP-890).
```

Comment:
Can we be a bit more specific here. Version 12 is for optimizing KIP-890 transactions. The produce request will also include an AddPartitionsToTxn call. etc. 

Ditto for the TxnOffsetCommit

Reply:
Done.